### PR TITLE
Adds front page.

### PR DESCRIPTION
**Pull request by chilic**

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -81,11 +84,12 @@ assetic:
                 - "%kernel.root_dir%/../vendor/owenversteeg/min/less/tables.less"
                 - "%kernel.root_dir%/../vendor/owenversteeg/min/less/messages.less"
             output: css/min.css
-
         common_css:
             inputs:
-              - "@WebsiteBundle/Resources/public/css/common.css"
+                - "bundles/website/css/common.css"
```
Comment (vkhramtsov) [inline app/config/config.yml]: Why did you change this line?

Comment (chilic) [inline app/config/config.yml]: @vkhramtsov I know it's little bit ugly, but in other way cssrewrite works not correct.

Comment (vkhramtsov) [inline app/config/config.yml]: Looks like you are using `css` from `webroot`. Strange. May I check this configuration later today, if you don't mind?

Comment (chilic) [inline app/config/config.yml]: @vkhramtsov sure! go ahead. I really need your help with this.

Comment (vkhramtsov) [inline app/config/config.yml]: May I change paths to images in css to make this work without css rewrite?

**Comment by vkhramtsov** [comment]
Unfortunately I cannot push code into @chilic repo, so let merge this code as is and I will fix it in next MR.
